roles/pulp_webserver/defaults/main.yml

```diff
@@ -11,3 +11,7 @@ pulp_webserver_httpd_servername: '{{ ansible_facts.fqdn }}'
 # https://pulp.plan.io/issues/5995
 pulp_webserver_static_dir: "{{ pulp_user_home | regex_replace('\\/$', '') }}/pulpcore_static"
 pulp_webserver_tls_files_remote: false
+
```

Comment:
In the future we may forget why we did this, a short comment here would be helpful

Reply:
@fao89 updated. Let me know if that is ok with you.

Reply:
thank you!